packages/editor-ui/src/stores/posthog.store.ts

```diff
@@ -161,10 +161,29 @@ export const usePostHog = defineStore('posthog', () => {
 		trackedDemoExp.value[name] = variant;
 	};
 
+	const capture = (event: string, properties: IDataObject) => {
+		if (typeof window.posthog?.capture === 'function') {
+			window.posthog.capture(event, properties);
+		}
+	};
+
+	const setMetadata = (metadata: IDataObject, target: 'user' | 'events') => {
+		if (typeof window.posthog?.people?.set !== 'function') return;
+		if (typeof window.posthog?.register !== 'function') return;
+
+		if (target === 'user') {
+			window.posthog?.people?.set(metadata);
+		} else if (target === 'events') {
+			window.posthog?.register(metadata);
+		}
+	};
+
 	return {
```

Comment:
We said to also bring the identity from the posthog-hooks but we are already identifying in this file! 

https://github.com/n8n-io/n8n/blob/4e3b8bdb1469366e1d36d20ef7741b9be44ccf55/packages/editor-ui/src/stores/posthog.store.ts#L71

Reply:
perfect..